theano/tensor/elemwise.py

```diff
@@ -455,16 +472,18 @@ class Elemwise(OpenMPOp):
     """
 
     __props__ = ("scalar_op", "inplace_pattern")
-
+    params_type = Generic()
     def __init__(self, scalar_op, inplace_pattern=None, name=None,
                  nfunc_spec=None, openmp=None):
+
         if inplace_pattern is None:
             inplace_pattern = frozendict({})
+
         self.name = name
         self.scalar_op = scalar_op
-        self.inplace_pattern = frozendict(inplace_pattern)
-        self.destroy_map = dict((o, [i]) for o, i in self.inplace_pattern.items())
```

Comment:
You must keep the destroy_map for the Theano execution engine. Otherwise, it won't execute the nodes in a good order.